lib/language_pack/ruby.rb

```diff
@@ -829,10 +829,24 @@ def load_bundler_cache
       rubygems_version_cache  = "rubygems_version"
       stack_cache             = "stack"
 
+      old_bundler_version  = @metadata.read(buildpack_version_cache)
```

Comment:
I merged this maybe 2 seconds before you left that comment. I'll update with a chomp.